CHANGELOG.md

```diff
@@ -9,6 +9,11 @@
 - [0.2.0](#020---20180924)
 - [0.1.0](#010---20180615)
 
+##  [0.4.1] - 2019/07/31
+
+- Fix issue where the plugin's shared dictionary would not be properly
+initialized
```

Comment:
Also, please add a link (for 0.4.1) to the diff between the last released version and this new one at the bottom of the changelog. It will be a 404 right now but will work once you push the tag.